### PR TITLE
Make fullscreen map available for desktop

### PR DESCRIPTION
**Pull request by LyraXaph**
Fix for #861, still working on tasks 3 and 4 (optional ones).

Had trouble with tests again, tried to work around it by manually updating snapshot files.

### CODE REVIEW

**Comment by tiltec** [comment]
Were the snapshot problems time-related?

**Comment by LyraXaph** [comment]
Yes I think so, same as last time.. (#872)

File: src/pages/Map.vue [hunk 1]
```diff
@@ -24,6 +25,17 @@ export default {
...
   },
+  data () {
+    return {
+      zoomLvl: null,
+    }
+  },
+  methods: {
+    mapZoomed (zoomLvl) {
+      this.$router.replace({ query: {zoom: zoomLvl} })
+      this.zoomLvl = Number(zoomLvl)
```
[the remark this answers is not in the document]
Comment (LyraXaph) [reply src/pages/Map.vue]: I was thinking of using it to pass the value to the map component. But I don't think its a good solution (still WIP).

**Comment by LyraXaph** [comment]
I think I am done with this, not sure if the checks in the StandardMap component that I added are the nicest solution tho.

File: src/components/Map/StandardMap.vue [hunk 2]
```diff
@@ -75,6 +76,14 @@ export default {
       type: Boolean,
       default: false,
     },
+    centerUrl: {
```
Comment (tiltec) [inline src/components/Map/StandardMap.vue]: I'd call those props (including the parent ones) `forceCenter` and `forceZoom` to make their behavior clearer. They don't carry a URL in them, after all.

File: src/components/Sidenav/SidenavMapUI.vue [hunk 3]
```diff
@@ -10,6 +10,22 @@
       slot="tools"
       class="tools"
     >
+      <q-btn
```
Comment (tiltec) [inline src/components/Sidenav/SidenavMapUI.vue]: Ah, I found out why the link doesn't work! It seems that nesting `router-link` inside `q-btn` doesn't work. It has to be the other way round. Unfortunately, this creates a problem where the icon would disappear on hovering, so I'd propose to add a `click` handler to `q-btn` and remove the `router-link` completely:

```
@click="$router.push({name: 'map', params: {groupId: currentGroup.id}})"
```

**Comment by tiltec** [comment]
I added the snapshots and rebased to master :)

**Pull request merged**